model/cluster_utility.go

```diff
@@ -0,0 +1,156 @@
+package model
```

Comment:
What about some small unit tests for this file?

Reply:
yep, makes sense, I'll write some tests

Reply:
tests for this file are in this commit https://github.com/mattermost/mattermost-cloud/pull/143/commits/5bd517e86af2583ee4f18f2c0c6bc860526778f9